gbpservice/nfp/scripts/mode_shift_enterprise.sh

```diff
@@ -2,27 +2,43 @@
 
```

Comment:
We will do that, once all the review is done. So that, theses changes won't lost.